src/goto-harness/common_harness_generator_options.h

```diff
@@ -16,6 +16,7 @@ Author: Diffblue Ltd.
 #define COMMON_HARNESS_GENERATOR_MAX_ARRAY_SIZE_OPT "max-array-size"
 #define COMMON_HARNESS_GENERATOR_FUNCTION_POINTER_CAN_BE_NULL_OPT              \
   "function-pointer-can-be-null"
+#define COMMON_HARNESS_GENERATOR_MEMBER_SELECTION_OPT "member-selection"
```

Comment:
I don’t really like the name `member-selection`, following the pattern it should be called something like `nondet-member` or `havoc-member`

Reply:
Done.